### ISSUE TITLE
CVE-2023-44487 in v0.16.2

### ISSUE THREAD

**Issue by gerbil** [aquasecurity/trivy-operator#1577]
**What steps did you take and what happened:**

High 7.5 cve was found by Aqua - https://nvd.nist.gov/vuln/detail/CVE-2023-44487

**Environment:**
- Trivy-Operator version v0.16.2

**Comment by chen-keinan** [comment]
Latest scan by trivy :
```

usr/local/bin/trivy-operator (gobinary)

Total: 2 (UNKNOWN: 0, LOW: 0, MEDIUM: 2, HIGH: 0, CRITICAL: 0)

┌──────────────────┬────────────────┬──────────┬────────┬───────────────────┬───────────────┬──────────────────────────────────────────────────────────────┐
│     Library      │ Vulnerability  │ Severity │ Status │ Installed Version │ Fixed Version │                            Title                             │
├──────────────────┼────────────────┼──────────┼────────┼───────────────────┼───────────────┼──────────────────────────────────────────────────────────────┤
│ golang.org/x/net │ CVE-2023-39325 │ MEDIUM   │ fixed  │ v0.16.0           │ 0.17.0        │ rapid stream resets can cause excessive work                 │
│                  │                │          │        │                   │               │ (CVE-2023-44487)                                             │
│                  │                │          │        │                   │               │ https://avd.aquasec.com/nvd/cve-2023-39325                   │
│                  ├────────────────┤          │        │                   │               ├──────────────────────────────────────────────────────────────┤
│                  │ CVE-2023-44487 │          │        │                   │               │ Multiple HTTP/2 enabled web servers are vulnerable to a DDoS │
│                  │                │          │        │                   │               │ attack (Rapid...                                             │
│                  │                │          │        │                   │               │ https://avd.aquasec.com/nvd/cve-2023-44487                   │
└──────────────────┴────────────────┴──────────┴────────┴───────────────────┴───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by chen-keinan** [comment]
Latest trivy-operator `v0.16.4`  version do not include this vulnerability:

```
trivy image ghcr.io/aquasecurity/trivy-operator:0.16.4
2023-11-14T10:36:33.529+0200	INFO	Need to update DB
2023-11-14T10:36:33.529+0200	INFO	DB Repository: ghcr.io/aquasecurity/trivy-db
2023-11-14T10:36:33.529+0200	INFO	Downloading DB...
40.75 MiB / 40.75 MiB [---------------------------------------------------------------------------------------------------------------------------------] 100.00% 13.39 MiB p/s 3.2s
2023-11-14T10:36:38.288+0200	INFO	Vulnerability scanning is enabled
2023-11-14T10:36:38.288+0200	INFO	Secret scanning is enabled
2023-11-14T10:36:38.288+0200	INFO	If your scanning is slow, please try '--scanners vuln' to disable secret scanning
2023-11-14T10:36:38.288+0200	INFO	Please see also https://aquasecurity.github.io/trivy/v0.47/docs/scanner/secret/#recommendation for faster secret detection
2023-11-14T10:36:42.399+0200	INFO	Detected OS: alpine
2023-11-14T10:36:42.399+0200	WARN	This OS version is not on the EOL list: alpine 3.19_alpha20230901
2023-11-14T10:36:42.399+0200	INFO	Detecting Alpine vulnerabilities...
2023-11-14T10:36:42.400+0200	INFO	Number of language-specific files: 1
2023-11-14T10:36:42.400+0200	INFO	Detecting gobinary vulnerabilities...

ghcr.io/aquasecurity/trivy-operator:0.16.4 (alpine 3.19_alpha20230901)

Total: 2 (UNKNOWN: 0, LOW: 0, MEDIUM: 2, HIGH: 0, CRITICAL: 0)

┌────────────┬───────────────┬──────────┬────────┬───────────────────┬───────────────┬───────────────────────────────────────────────────────┐
│  Library   │ Vulnerability │ Severity │ Status │ Installed Version │ Fixed Version │                         Title                         │
├────────────┼───────────────┼──────────┼────────┼───────────────────┼───────────────┼───────────────────────────────────────────────────────┤
│ libcrypto3 │ CVE-2023-5678 │ MEDIUM   │ fixed  │ 3.1.4-r0          │ 3.1.4-r1      │ openssl: Generating excessively long X9.42 DH keys or │
│            │               │          │        │                   │               │ checking excessively long X9.42...                    │
│            │               │          │        │                   │               │ https://avd.aquasec.com/nvd/cve-2023-5678             │
├────────────┤               │          │        │                   │               │                                                       │
│ libssl3    │               │          │        │                   │               │                                                       │
│            │               │          │        │                   │               │                                                       │
│            │               │          │        │                   │               │                                                       │
└────────────┴───────────────┴──────────┴────────┴───────────────────┴───────────────┴───────────────────────────────────────────────────────┘

usr/local/bin/trivy-operator (gobinary)

Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 1, HIGH: 0, CRITICAL: 0)

┌──────────────────────────┬─────────────────────┬──────────┬────────┬──────────────────────┬───────────────┬────────────────────────────────────────────────────────┐
│         Library          │    Vulnerability    │ Severity │ Status │  Installed Version   │ Fixed Version │                         Title                          │
├──────────────────────────┼─────────────────────┼──────────┼────────┼──────────────────────┼───────────────┼────────────────────────────────────────────────────────┤
│ github.com/docker/docker │ GHSA-jq35-85cj-fj4p │ MEDIUM   │ fixed  │ v24.0.5+incompatible │ 24.0.7        │ /sys/devices/virtual/powercap accessible by default to │
│                          │                     │          │        │                      │               │ containers                                             │
│                          │                     │          │        │                      │               │ https://github.com/advisories/GHSA-jq35-85cj-fj4p      │
└──────────────────────────┴─────────────────────┴──────────┴────────┴──────────────────────┴───────────────┴────────────────────────────────────────────────────────┘
```
